TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -939,12 +939,56 @@ public class TrafficManagerTool
                     }
                 }
 
-                return HoveredNodeId != 0 || HoveredSegmentId != 0;
+                if (HoveredNodeId != 0) {
```

Comment:
Argument can be made that his should be done at line 877 when we determine HoveredNodeID. That approach works for all cases except for level crossings. I found no way to get the following function to find level crossing nodes:
```RayCast(segmentInput, out RaycastOutput segmentOutput) ```
Therefore I decided to call `GetHoveredSegmentFromNode()` here.

The code can take multiple paths. the most complicated one is this:
1. We determine the `HoveredSegmentID`.
2. We determine `HoveredNodeID` by calculating shortest distance to the segment start or end node.
3. We fix `HoveredSegmentID` again by finding the segment which makes the minimum angle with the mouse-to-node vector.

it may sound confusing but thats the only way to support level crossings.